backend/utils/graph_db_utils_test.py

```diff
@@ -0,0 +1,64 @@
+from unittest.mock import MagicMock
+from neo4j import Driver, Session
+import pytest
+# We assign an alias to "test_connection" to avoid pytest treating it as another test function
+from utils.graph_db_utils import test_connection as verify_connection, create_goal
```

Comment:
I don't believe you need the `.graph_db_utils` as we have mentioned these methods in the __init__ file
```suggestion
from utils import test_connection as verify_connection, create_goal
```

Reply:
Thanks @Ch0live - that works. 👍  Updated!